common/cards/advent-of-tcg/single-use/wind-burst.ts

```diff
@@ -23,7 +20,7 @@ const WindBurst: SingleUse = {
 	rarity: 'common',
 	tokens: 0,
 	description:
-		'After your attack, you and your opponent must move your active Hermit and any attached cards to an open row on the game board if able.\nYour opponent moves their active Hermit first.',
+		"After your attack, choose an open row on the game board to move your opponent's active Hermit and any attached cards, if possible. Then choose an open row on the game board if able, to move your active Hermit and any attached cards.",
```

Comment:
I think this description is a little clearer.
```suggestion
		"After your attack, choose an open row on the game board to move your opponent's active Hermit and any attached cards, if possible. Then, if possible, choose an open row on the game board to move your active Hermit and any attached cards.",
```